templates/etc/monit/exec-scripts/smpp-connection-alerts.sh.j2

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+ipsec_failure_file=/tmp/ipsec-failure-time
+# If failed
+if [[ "$1" == "failed" ]] ; then
+    # write the first failure incident to file
+    echo "$MONIT_DATE" > "$ipsec_failure_file"
+    # Send first incident occurrence alert
+    # Create Opsgenie Incident and save request ID to file
+    source /etc/monit/exec-scripts/opsgenie-alert
```

Comment:
Shoud this be `source /etc/monit/exec-scripts/smpp-connection-opsgenie-alerts.sh` instead?